pkg/util/etcdutil/member.go

```diff
@@ -126,3 +127,19 @@ func (ms MemberSet) ClientURLs() []string {
 	}
 	return endpoints
 }
+
+func GetIDFromName(name string) (int, error) {
```

Comment:
GetIDFromMemberName.

I am not sure we should call it ID. each member has its etcd member ID. this ID is confusing.